scripts/misc/import_oldhgmd.pl

```diff
@@ -65,73 +87,93 @@
 }
 
 sub test {
-  debug($config, "Running in test mode"); 
 
-  debug($config, "Dumping old variation tables with HGMD as source from old database");
+  debug($config, "Running in test mode"); 
+  
+  my $old_dbname = $old_dbh->dbname;
+  debug($config, "Dumping old phenotype data with HGMD as source from $old_dbname");
   dump_vdata_into_file($old_dbh);
 
-  debug($config, "Dumping old phenotype with HGMD as source from old database");
+  debug($config, "Dumping old phenotype data with HGMD as source from $old_dbname");
   dump_pheno($old_dbh);
 
   debug($config, "Sorting files based on the variation_id column");
   $TMP_DIR = $TMP_DIR . "/";
   system("sort -k 1 -o ${TMP_DIR}${sorted_old_var} ${TMP_DIR}${old_var_file}");
   system("sort -k 6 -o ${TMP_DIR}${sorted_old_var_feat} ${TMP_DIR}${old_vf_file}");
 
+  if ($config->{transcript}) {
+    debug($config, "Sorting files based on the variation_feature_id column");
```

Comment:
This message should be specific to TV:
`debug($config, "Sorting transcript variation files based on the variation_feature_id column");`